File: Competetive programming/Python/Unit 3/Automated_Judge_Script.py
print("===== AUTOMATED JUDGE SCRIPT =====\n")

correct = input("Enter Correct Output: ")

output = input("Enter User Output: ")

if correct == output:

    print("\nAccepted")

else:

    num1 = ""
    num2 = ""

    for c in correct:

        if c.isdigit():

            num1 += c

    for c in output:

        if c.isdigit():

            num2 += c

    if num1 == num2:

        print("\nPresentation Error")

    else:

        print("\nWrong Answer")

File: Competetive programming/C++/Unit 3/Automated_Judge_Script.cpp
#include <iostream>
#include <string>
#include <cctype>
using namespace std;

int main() {

    string correct, output;

    cout << "Enter Correct Output: ";
    getline(cin, correct);

    cout << "Enter User Output: ";
    getline(cin, output);

    if(correct == output) {
        cout << "\nAccepted";
    }
    else {

        string num1 = "", num2 = "";

        for(char c : correct)
            if(isdigit(c))
                num1 += c;

        for(char c : output)
            if(isdigit(c))
                num2 += c;

        if(num1 == num2)
            cout << "\nPresentation Error";
        else
            cout << "\nWrong Answer";
    }

    return 0;
}